clustered/client/src/main/java/org/ehcache/clustered/client/config/ClusteringServiceConfiguration.java

```diff
@@ -60,6 +60,14 @@ public ClusteringServiceConfiguration(URI clusterUri) {
     this(clusterUri, Timeouts.DEFAULT);
   }
 
+  /**
+   * Creates a {@code ClusteringServiceConfiguration} from the properties provided.
+   *
+   * @param servers the non-{@code null} Iterable<InetSocketAddress> identifying the servers in the cluster
+   * @param clusterTierManager the non-{@code null} clusterTierManager identifying the cache manager
```

Comment:
This (and the other instances) is a little too close to referring to the cache manager name.

Reply:
Changed.